docs/csharp/misc/cs5001.md

```diff
@@ -1,17 +1,17 @@
 ---
 title: "Compiler Error CS5001"
 ms.date: 08/27/2018
-f1_keywords: 
+f1_keywords:
   - "CS5001"
-helpviewer_keywords: 
+helpviewer_keywords:
   - "CS5001"
 ms.assetid: e1e26e75-84e0-47c7-be8a-3c4fd0d6f497
 ---
 # Compiler Error CS5001
 
-Program 'program' does not contain a static 'Main' method suitable for an entry point  
-  
-This error occurs when no static `Main` method with a correct signature is found in the code that produces an executable file. This error also occurs if the entry point function, `Main`, is defined with the wrong case, such as lower-case `main`. For information about the rules that apply to the `Main` method, see [Main() and Command-Line Arguments](../programming-guide/main-and-command-args/index.md).
+Program does not contain a static 'Main' method suitable for an entry point
```

Comment:
I think the error message contains the name of the program in the middle of the sentence, doesn't it? Let me double check.

If it does, we should keep the original version.

Reply:
I didn't generate the error by myself, but have checked Roslyn repo and didn't find it contains the program name.

Reply:
https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Portable/CSharpResources.resx#L3779

Reply:
Yep. I've created a quick test and also confirmed that. Thanks!

Reply:
I've tried in Visual Studio and program name isn't included.

![image](https://user-images.githubusercontent.com/31348972/65836775-5eaefc80-e2f2-11e9-9eb3-f2838dc46452.png)

However, In some older versions of C# compiler, (not sure if it's even before Roslyn or not), the program name **was** included.